pandas/core/generic.py

```diff
@@ -5201,60 +5201,209 @@ def abs(self):
+        same as the median.
 
-        Notes
-        -----
-        The output DataFrame index depends on the requested dtypes:
+        For ``object`` data (e.g. strings or timestamps), the result's index
+        will include ``count``, ``unique``, ``top``, and ``freq``. The ``top``
+        is the most common value. The ``freq`` is the most common value's
+        frequency. Timestamps also include the ``first`` and ``last`` items.
 
-        For numeric dtypes, it will include: count, mean, std, min,
-        max, and lower, 50, and upper percentiles.
+        If multiple ``object`` values have the highest count, then the
+        ``count`` and ``top`` results will be arbitrarily chosen from
+        among those with the highest count.
 
-        For object dtypes (e.g. timestamps or strings), the index
-        will include the count, unique, most common, and frequency of the
-        most common. Timestamps also include the first and last items.
+        For mixed data types provided via a `DataFrame`, the result will
+        include a union of attributes of each type.
 
-        For mixed dtypes, the index will be the union of the corresponding
-        output types. Non-applicable entries will be filled with NaN.
-        Note that mixed-dtype outputs can only be returned from mixed-dtype
-        inputs and appropriate use of the include/exclude arguments.
+        The `include` and `exclude` parameters can be used to limit
+        which columns in a `DataFrame` are analyzed for the output.
+        The parameters are ignored when analyzing a `Series`.
 
-        If multiple values have the highest count, then the
-        `count` and `most common` pair will be arbitrarily chosen from
-        among those with the highest count.
+        Parameters
+        ----------
+        percentiles : list of numbers, optional
+            The percentiles to include in the output. All should
+            fall between 0 and 1. The default is
+            ``[.25, .5, .75]``, which returns the 25th, 50th, and
+            75th percentiles.
+        include : None (default), 'all', or list of dtypes or strings, optional
+            A white list of data types to include in the result. Ignored
+            for `Series`. Here are the options:
+
+            - None (default). The result will include all ``numeric`` columns.
+            - 'all'. All columns on the input will be included in the output.
+            - A list of dtypes or strings. Limits the results to the
+              provided data types.
+              To limit the result to numeric types submit
+              ``numpy.number``. To limit it instead to categorical
+              objects submit the data type ``object``. Strings
+              can also be used in the style of
+              `select_dtypes` (e.g. df.describe(include=['O']))
+        exclude : None (default) or a list of dtypes or strings, optional,
```

Comment:
I agree this is a balancing act. I decided to separate them after consulting the source code, where I found that include and exclude actually have slightly different behavior. From what I can tell, `all` is not an acceptable input for exclude. Separating them also allowed me space to try to write more explicit descriptions that contrast the two keywords.